cpp/CMakeLists.txt

```diff
@@ -539,7 +531,8 @@ target_compile_options(
                "$<$<COMPILE_LANGUAGE:CUDA>:${CUVS_CUDA_FLAGS}>"
 )
 # ensure CUDA symbols aren't relocated to the middle of the debug build binaries
-target_link_options(cuvs PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/fatbin.ld")
+#
```

Comment:
Was this supposed to be commenting this out? Should this be removed?

Reply:
I commented this out initially, but Robert helped me to tweak it back.